massa-async-pool/src/pool.rs

```diff
@@ -1035,3 +1035,374 @@ impl AsyncPool {
         db.delete_key(batch, can_be_executed_key!(serialized_message_id));
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::str::FromStr;
+    use std::sync::Arc;
+
+    use massa_db_exports::{MassaDBConfig, MassaDBController};
+    use massa_models::config::{
+        MAX_ASYNC_POOL_LENGTH, MAX_DATASTORE_KEY_LENGTH, MAX_FUNCTION_NAME_LENGTH,
+        MAX_PARAMETERS_SIZE, THREAD_COUNT,
+    };
+    use massa_models::{address::Address, amount::Amount, slot::Slot};
+
+    use crate::message::AsyncMessageTrigger;
+
+    use massa_db_worker::MassaDB;
+    use parking_lot::RwLock;
+    use tempfile::tempdir;
+
+    use super::*;
+
+    fn dump_column(
+        db: Arc<RwLock<Box<dyn MassaDBController>>>,
+        column: &str,
+    ) -> BTreeMap<Vec<u8>, Vec<u8>> {
+        db.read()
+            .iterator_cf(column, MassaIteratorMode::Start)
+            // .collect::<BTreeMap<Vec<u8>, Vec<u8>>>()
+            .collect()
+    }
+
+    fn get_message() -> AsyncMessage {
```

Comment:
Small code duplication here, no? Don't change it if it's the only duplicated helper function though.